samples/hello_world/hello_world.yaml

```diff
@@ -0,0 +1,19 @@
+description:
+    name: hello_world
+    description: A simple 'Hello World' study.
+
+env:
+    variables:
+        OUTPUT_PATH: ./sample_output/hello_world
+
+study:
+<<<<<<< HEAD
+    - name: hello_world
+      description: Say hello to the world!
+=======
+    - name: say_hi
+      description: Build the serial version of LULESH.
+>>>>>>> a435e36481aee4f6c6044a9932d459e122b58a9b
```

Comment:
Looks like a merge conflict was accidentally committed.

Reply:
Just pushed the fix for this. I did a search and no other missed merges came up.